Source/CarthageKit/Xcode.swift

```diff
@@ -97,7 +124,21 @@ public struct BuildArguments {
 	public var configuration: String?
 
 	/// The platform SDK to build for.
-	public var sdk: SDK?
+	public var sdk: SDK? {
+		didSet {
+			// If SDK is the iOS simulator, then also set the destination.
+			// This fixes problems when the project deployment version is lower than the target's
+			// and includes simulators unsupported by the target.
+			// Example: Target is at 8.0, project at 7.0, xcodebuild chooses the first
+			// simulator on the list, iPad 2 7.1, which is invalid for the target.
+			if sdk == .iPhoneSimulator {
+				self.destination = BuildDestination.iOSSimulator(name: "iPhone 6", OS: "latest")
```

Comment:
Yeah, I think you're right. I was not sure here (first time digging trough Carthage) so I put it in the setter, but will move it :+1: